cas-server-webapp/src/main/webapp/WEB-INF/cas.properties

```diff
@@ -691,3 +691,8 @@ accept.authn.users=casuser::Mellon
 # cas.pac4j.oidc.customParamValue1=
 # cas.pac4j.oidc.customParamKey2=
 # cas.pac4j.oidc.customParamValue2=
+
+##
+# CAS OAuth server support: keys are mandatory for JWT support
+cas.oauth.jwt.signingSecret=
```

Comment:
Also, this `keys are mandatory for JWT support` comment might be also misleading. I think you should clarify that keys are required for access tokens, since we also use JWT for token authN.